OpenRobertaServer/src/main/java/de/fhg/iais/roberta/javaServer/restServices/all/controller/ClientProgramController.java

```diff
@@ -615,7 +615,12 @@ public Response getProgramsFromGallery(@OraData DbSession dbSession, JSONObject
             HttpSessionState httpSessionState = UtilForREST.handleRequestInit(LOG, request, true);
             ProgramProcessor programProcessor = new ProgramProcessor(dbSession, httpSessionState);
             int userId = httpSessionState.getUserId();
-            JSONArray programInfo = programProcessor.getProgramGallery(userId);
+            JSONObject data = request.getJSONObject("data");
+            String group = "";
+            if(data.has("group")){
```

Comment:
this line is not formatted, do you have installed the roberta javascript formatter?